### DISCUSSION TITLE
The second step of calculation always displays an error. I followed the instructions completely, I don't know which step went wrong

### DISCUSSION THREAD

**Discussion by jiaziqi144** [Anura3D/Anura3D_OpenSource#193] [Q&A]
![微信截图_20240305134907](https://github.com/Anura3D/Anura3D_OpenSource/assets/162278278/f656c6f9-495c-4b02-aee8-b06ea05999e0)

**Comment by EmreDumlu-UM** [comment]
Hi,
Would you like share paraview views to understand what's going on?
